conf/messages

```diff
@@ -145,7 +145,8 @@ international.addressLookup.homeAddress.editPage.line1Label                = Add
 international.addressLookup.homeAddress.editPage.line2Label                = Address line 2
 international.addressLookup.homeAddress.editPage.line3Label                = Address line 3 (optional)
 international.addressLookup.homeAddress.editPage.townLabel                 = Town/City
-international.addressLookup.homeAddress.editPage.postcodeLabel             = Postcode (optional)
+international.addressLookup.homeAddress.editPage.postcodeLabel             = Postcode
```

Comment:
Do we need to be updating ALF content? In fact we do not even send postcodeHints there.